Synergism.css

```diff
@@ -741,10 +749,33 @@ nav li:hover {
     margin-bottom: 3px;
 }
 
+.themeButton {
+    padding: 5px;
+    margin: 1px;
+    transition-duration: 0.15s;
+}
+
+.themeButton:hover,
 .promocodeButton:hover {
     background-color: var(--hover-color);
 }
 
+.themeWindow {
+    display: none;
+    background-color: var(--box-color);
+    border: 3px solid;
+    border-color: var(--boxmain-bordercolor);
+    margin-left: 10px;
+}
+
+.themeWindow:hover,
+#theme:focus + .themeWindow {
+    display: inline-grid;
+    grid-template-columns: repeat(2, 150px);
+    padding: 2px;
+    position: absolute;
```

Comment:
absolute positioned

Reply:
Yes because its a pop up
Same as if you would click your right button on mouse
(That one is absolute positioned as well)